### PR TITLE
ENH: update metadata format description to include comments, blank lines, and QIIME 1 support

### PR DESCRIPTION
**Pull request by jairideout**
Also reworked some of the text for clarity, added more details in a few places, and fixed a typo.

Addresses part of #173.

### CODE REVIEW

File: source/tutorials/metadata.rst [hunk 1]
```diff
@@ -5,40 +5,42 @@ Metadata in QIIME 2
 
 Metadata provides the key to gaining biological insight from your raw data. In QIIME 2, sample metadata includes technical details, such as the DNA barcodes that were used for each sample, or descriptions of the samples, such as which subject, time point and body site each sample came from in a human microbiome time series. Feature metadata is often a feature annotation, such as the taxonomy assigned to a sequence variant. Sample and feature metadata are used by many methods and visualizers in QIIME 2.
 
-.. qiime1-users:: In QIIME 1, metadata (also known as the *metadata mapping file*) was a user-defined TSV file that contained these study-specific fields. QIIME 2 expands upon this idea, allowing users to provide their own study metadata via a TSV file or by viewing QIIME 2 artifacts as metadata. Examples of both are presented in the following sections. Begin by creating a directory to conduct the work in:
```
Comment (jairideout) [inline source/tutorials/metadata.rst]: I removed redundant *Begin by creating a directory to conduct the work in:* from this block.

File: source/tutorials/metadata.rst [hunk 2]
```diff
@@ -5,40 +5,42 @@ Metadata in QIIME 2
...
-
-Begin by creating a directory to conduct the work in:
-
-.. command-block::
```
Comment (jairideout) [inline source/tutorials/metadata.rst]: I moved this command-block down in the tutorial where we have users download an example TSV file. In its previous location, it wasn't clear why we're having users create a directory, when the actual work is done later in the tutorial.

Comment (ebolyen) [inline source/tutorials/metadata.rst]: :+1:

File: source/tutorials/metadata.rst (hunk 1 above)
Comment (thermokarst) [inline source/tutorials/metadata.rst]: Oops, that was almost certainly copy-paste-errors-dot-com. Good catch @jairideout!

**Pull request merged**